.github/workflows/antsibull-build.yml

```diff
@@ -29,21 +29,25 @@ jobs:
             python: '3.9'
             antsibull_changelog_ref: 0.24.0
             antsibull_core_ref: stable-2
+            antsibull_docutils_ref: 1.0.0  # isn't used by antsibull-changelog 0.24.0
```

Comment:
Should we set this to false and then add an `if: matrix.antsibull_docutils_ref` to the step that clones it?

Reply:
I'm not sure you can have data types other than strings here, but `""` will also do :)

Reply:
I tried that in 9184d3dd62d425e7b081cfd9009158288d3f666b and apparently screwed up the workflow syntax. But while doing that I noticed another problem, described in https://github.com/ansible-community/antsibull/pull/612#issuecomment-2305617932: `OTHER_ANTSIBULL_MODE=local` requires that antsibull-docutils is around.